### ISSUE TITLE
 Error in MikuMikuModel export

### ISSUE THREAD

**Issue by wuwuhonghong** [blueskythlikesclouds/MikuMikuLibrary#48]
Please answer if you know what the problem is, it will help me a lot, because it is very important to me, thank you very much!
I have some problems in the process of using MikuMikuModel: the model can only be exported in dae format, and the texture cannot be exported.
In addition to the model can be exported, other operations are the following errors:
![46JB3 U}Q3W 3SN_ 8GC5E](https://user-images.githubusercontent.com/130895540/232287317-683e3bb4-3e17-4e67-a1f6-48358d235e70.png)

**Comment by blueskythlikesclouds** [comment]
Did you install the vcredist runtime?

Install both of these and try again:
[https://aka.ms/vs/16/release/VC_redist.x64.exe](https://aka.ms/vs/16/release/VC_redist.x64.exe)
[https://aka.ms/vs/16/release/VC_redist.x86.exe](https://aka.ms/vs/16/release/VC_redist.x86.exe)

**Comment by wuwuhonghong** [comment]
> 
Unable to install Microsoft .NET Framework
Because a later version is already installed

**Comment by wuwuhonghong** [comment]
> 

Yes, I have installed everything I need, but I still get an error. This time I can’t even export the model. I was able to export the model before.

**Comment by DGCK81LNN** [comment]
I'm on Windows 11 (25H2) and have the same problem, unable to view textures or export FBX. Exporting .dae partially works but still errors and the textures are missing.

I tried installing .NET runtime and VC++ runtimes, both say I already have a newer version installed. Removing my newer VC++ and installing using the link in Releases didn't help either.
